jerry-core/ecma/builtin-objects/ecma-builtin-array-prototype.c

```diff
@@ -869,6 +869,17 @@ ecma_builtin_array_prototype_object_slice (ecma_value_t arg1, /**< start */
 
     if (ext_from_obj_p->u.array.u.hole_count < ECMA_FAST_ARRAY_HOLE_ONE)
     {
+      ecma_value_t *from_buffer_p = ECMA_GET_POINTER (ecma_value_t, obj_p->u1.property_list_cp);
+
+      if (JERRY_UNLIKELY (from_buffer_p == NULL))
```

Comment:
How about check whether the `obj_p->u1.property_list_cp == JMEM_CP_NULL`, then leave the  `ECMA_GET_NON_NULL_POINTER `?

Reply:
True, the `..GET_POINTER` does a check already, we can eliminate the double check.